exercises/word-count/word_count_test.py

```diff
@@ -41,6 +41,21 @@ def test_mixed_case(self):
             sorted(list(word_count('go Go GO Stop stop').values()))
         )
 
+    # def test_apostrophes(self):
+    #     self.assertEqual(
+    #         {'first': 1, "don't": 2, 'laugh': 1, 'then': 1, 'cry': 1},
+    #         word_count("First: don't laugh. Then: don't cry.")
```

Comment:
Can you update the order of these arguments (and the ones in the rest of the test-suite) to match the `assertEqual(actual, expected)` format?